src/components/AccountNamesTable.vue

```diff
@@ -24,7 +24,7 @@
     </thead>
     <tbody>
       <tr
-        v-for="name in accountNames?.data"
+        v-for="name in accountNames.data"
```

Comment:
added some cleanup. THe optional chaining is not needed anymore as we have loaders